stl/inc/system_error

```diff
@@ -54,10 +54,11 @@ _INLINE_VAR constexpr bool is_error_condition_enum_v = is_error_condition_enum<_
 
 _EXPORT_STD class error_code;
 _EXPORT_STD class error_condition;
-_EXPORT_STD _NODISCARD error_code make_error_code(errc) noexcept;
-_EXPORT_STD _NODISCARD error_code make_error_code(io_errc) noexcept;
-_EXPORT_STD _NODISCARD error_condition make_error_condition(errc) noexcept;
-_EXPORT_STD _NODISCARD error_condition make_error_condition(io_errc) noexcept;

```

Comment:
No change requested: This is correct because both these removed declarations and the preserved definitions below are marked with `_EXPORT_STD`, so we aren't losing any exports.